include/tvm/runtime/vm/vm.h

```diff
@@ -270,6 +273,15 @@ class VirtualMachine : public runtime::ModuleNode {
    */
   void SetInput(std::string name, TVMArgs args, int offset);
 
+  /*!
+   * \brief Set one input tensor with index or name to a function.
+   * \param name The function name
+   * \param args args[1:] are two arguments (index or name, tensor) to the
```

Comment:
nit: might as well just pass two args since fixed size
nit: Would call it just SetOneInput to match above.

Reply:
if I correctly understood your first tip it was done